apps/s_client.c

```diff
@@ -3467,4 +3509,25 @@ static int ldap_ExtendedResponse_parse(const char *buf, long rem)
     return ret;
 }
 
+static char *base64encode (const void *buf, size_t len) {
+
+    /* BASE64 encoder - used only for encoding basic proxy authentication credentials
+     * Based on an idea by schulwitz here: 
+     * https://stackoverflow.com/questions/342409/how-do-i-base64-encode-decode-in-c/
+     */
+
+     BIO *b64_bio, *mem_bio;
+     BUF_MEM *mem_bio_mem_ptr;
+     b64_bio = BIO_new(BIO_f_base64());
```

Comment:
Hoist the assignments up into the declarations and put a blank line after all the de clarations.

Reply:
Done!